Containers/apache/healthcheck.sh

```diff
@@ -0,0 +1,7 @@
+if [ "$APACHE_PORT" != '443' ]; then
+    curl -skfI localhost:8000 || exit 1
+    curl -skfI localhost:$APACHE_PORT || exit 1
+else
```

Comment:
```suggestion
    curl -skfI localhost:$APACHE_PORT || exit 1
    curl -skfI $NC_DOMAIN:443 || exit 1
else
```
I am thinking if this should be also added

Reply:
this will not work in all cases so I wouldn't do this...